at-hoangtran/Week3/Day2/Exercise3.rb

```diff
@@ -0,0 +1,25 @@
+#Hold data
+lecturers = [
+  {lecturer: 'Paul Barry', subject:  ['Programming', 'Networking', 'Security', 'Open Source', 'Frameworks']},
+  {lecturer: 'Chris Meudec', subject:  ['Testing', 'Safety Systems', 'Formal Systems', 'Programming Languages']},
+  {lecturer: 'Nigel Whyte', subject:  ['Graphics', 'Imaging', 'Programming', 'Sign Language', 'Trees']},
+  {lecturer: 'Austin Kinsella', subject:  ['Networks', 'WANs', 'Programming', 'Macintosh', 'Digital Photography']},
+  {lecturer: 'Gerry Moloney', subject:  ['Placement', 'Employment', 'Emerging Systems', 'Web Development']},
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/b72baa980a083c5128cdafdaa21936228c8c182b/at-hoangtran/Week3/Day2/Exercise3.rb#L7"></span>Style/TrailingCommaInArrayLiteral: Avoid comma after the last item of an array. (https://github.com/rubocop-hq/ruby-style-guide#no-trailing-array-commas)</td>
    </tr>
  </tbody>
</table>